include/base.php

```diff
@@ -188,7 +188,7 @@ public function should_customize_menu_be_removed() {
 		}
 
 		$customize_register_hooks = count( array_merge( ...array_values( $wp_filter['customize_register']->callbacks ) ) );
-		if ( $customize_register_hooks > 1 ) {
+		if ( $customize_register_hooks > 1 + (int) ( 'page' === get_option( 'show_on_front' ) ) ) { // Are there other hooks than our own?
```

Comment:
Maybe we could add a comment similar to this:
```suggestion
		/*
		 * 'customize_register' is hooked by:
		 * @see PLL_Nav_Menu::create_nav_menu_locations()
		 * @see PLL_Frontend_Static_Pages::filter_customizer()
		 */
		if ( $customize_register_hooks > 1 + (int) ( 'page' === get_option( 'show_on_front' ) ) ) { // Are there other hooks than our own?
```